app/assets/javascripts/omni-nav-v2.js

```diff
@@ -446,11 +445,33 @@ var OmniNav2 = (function() {
       }
 
       var onSearchEnter = function(e) {
+        // The autocomplete container is not present until the user starts typing
+        // Add click event listener the first time it shows up
+        bindAutocomplete();
+
         if((e.type == 'click' || e.which == ENTER_KEY) && gcsElement.getInputQuery().length > 0) {
           showSearchResults();
         }
       }
 
+      var bindAutocomplete = function() {
+        // If both containers are bound, don't execute again
+        if(primaryAutocompleteBound && utilityAutocompleteBound) return;
+
+        if($element[0].className.includes("utility") && !utilityAutocompleteBound) {
+          $("table.gstl_50.gssb_c").find(".gsc-completion-container").on("click", onAutocompleteClick);
+          utilityAutocompleteBound = true;
+        }
+        else if(!primaryAutocompleteBound) {
+          $("table.gstl_51.gssb_c").find(".gsc-completion-container").on("click", onAutocompleteClick);
+          primaryAutocompleteBound = true;
+        }
+      }
+
+      var onAutocompleteClick = function(e) {
```

Comment:
Added a story to address a minor lingering issue with autocomplete: https://trello.com/c/rQcWMSLg